cd_on_quit/cd_on_quit.sh

```
@@ -1,15 +1,7 @@
-    # Linux
-    if [[ "$os" == "Linux" ]]; then
-        export SPF_LAST_DIR="${XDG_STATE_HOME:-$HOME/.local/state}/superfile/lastdir"
-    fi
-
-    # macOS
-    if [[ "$os" == "Darwin" ]]; then
-        export SPF_LAST_DIR="$HOME/Library/Application Support/superfile/lastdir"
-    fi
+    # Ask the binary where it writes the lastdir file, so this works for every
+    # installation method (Homebrew, package managers, manual install, ...)
+    export SPF_LAST_DIR="$(command spf path-list --lastdir-file)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)